src/tito/common.py

```diff
@@ -229,20 +229,31 @@ def create_builder(package_name, build_tag,
     return builder
 
 
-def find_file_with_extension(in_dir=None, suffix=None):
+def find_file_with_extension(in_dir=None, suffix=None, top=True):
```

Comment:
Unless I'm missing something I don't think the "top" param is actually used anywhere.